examples/Finding_Local_Peaks_in_Solar_Data.py

```diff
@@ -0,0 +1,76 @@
+"""
+=================================
+Finding Local Peaks in Solar Data
+=================================
+
+How to find regions of local maxima
+"""
+import astropy.units as u
+from skimage.feature import peak_local_max
+import matplotlib.pyplot as plt
+from mpl_toolkits import mplot3d
+from mpl_toolkits.mplot3d import Axes3D
+import numpy as np
+
+import sunpy.map
+from sunpy.data.sample import AIA_193_IMAGE
+
+###############################################################################
+# We will first create a Map using some sample data and display it.
+
+aiamap = sunpy.map.Map(AIA_193_IMAGE)
+plt.figure()
+aiamap.plot()
+plt.colorbar()
+
+
+###############################################################################
+# Before we find regions of local maxima, we need to create some variables that
+# store pixel coordinates for the 2D SDO/AIA data we are using.
+# These variables are used for plotting in 3D later on.
+
+x = y = np.arange(0, len(aiamap.data))
+X, Y = np.meshgrid(x, y)
+
+
+###############################################################################
+# We will only consider peaks within the AIA data that are above a threshold
+# value. Once these values are thresholded, the next step is to
+# calcualte the pixel locations of local maxima posistions.
+# This function comes from sci-kit image and the documenation is found
+# here `~skimage.feature.peak_local_max`.
+
+coordinates = peak_local_max(aiamap.data, min_distance=60, threshold_rel=0.2)
```

Comment:
A brief description of the values here might be worthwhile. Just to make it clear to a reader why these values were chosen or how they might have to be modified for other data.